### DISCUSSION TITLE
No Abaqus->LibMesh mapping information for ElemType QUAD8!

### DISCUSSION THREAD

**Discussion by etauq** [idaholab/moose#17639] [Q&A Meshing] [answered]
Hi,

I was planning to manually generate a mesh in ABAQUS mesh format. The elements should be second order elements, like CPE8 in ABAQUS (QUAD8 In libmesh). However, when I ran the simulation, there was an error "No Abaqus->LibMesh mapping information for ElemType QUAD8!". Does anyone know how to generate mesh with second order elements that MOOSE can read? Or does anyone know how to use 3x3 quadrature rule for CPE4 (QUAD4) elements in MOOSE?

Thanks,
Jack

**Comment by GregVernon** [comment, marked as answer]
There should be a QUAD9 in libMesh, but I don't think Abaqus supports QUAD9.  If you're in a pinch, however, Cubit can export QUAD9 elements via Exodus which should be supported by libMesh.  You could test with the free non-commercial license of Coreform Cubit: https://coreform.com/products/coreform-cubit/free-meshing-software/

That being said, it certainly seems like the QUAD8 should be supported, so this seems like a bug.  But generally speaking, I think you can expect Exodus files to have better support in libMesh than Abaqus files (@permcody, true statement?).

**Reply by etauq** [reply]
Thanks, I used Cubit to create the Exodus mesh. It works great!
